comet/utils/redis.py

```
@@ -0,0 +1,215 @@
+        try:
+            if isinstance(value, (dict, list, tuple)):
+                value = json.dumps(value)
+            elif not isinstance(value, str):
+                value = str(value)
+            
+            if ttl and ttl > 0:
+                await self.redis.setex(key, ttl, value)
+            else:
+                await self.redis.set(key, value)
+            
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)